### DISCUSSION TITLE
Which Asus services can be removed permanently?

### DISCUSSION THREAD

**Discussion by AreyUA** [seerge/g-helper#433] [General]
And how to remove and which Asus services can be safely removed? Tell me please. I want to say goodbye forever to all Asus services, but I'm afraid to break something (((

**Comment by seerge** [comment]
@AreyUA if hotkeys work for you as they should with asus optimization stopped - then yes , you can stop / disable it in services :)

**Reply by AreyUA** [reply]
I did not turn off any services when I click on M4, I am prompted to install Armory. I made this post to explain to me which services can be disabled on the 2021 model

**Reply by AreyUA** [reply]
maybe there is somewhere a list of services that can be found through the search for example and immediately disable

**Reply by seerge** [reply]
You can run https://github.com/seerge/g-helper/blob/main/debloat.bat and it will disable all main stuff, except Asus optimization. That one you need to disable by hand (for now), as i'm not sure if hotkeys work ok on all models

**Reply by AreyUA** [reply]
Thank you, what if something goes wrong? Is there a way to turn it on later?

**Reply by seerge** [reply]
Just start all services again. Nothing is being deleted there.

**Reply by AreyUA** [reply]
I launched the file, it seems that at first glance everything works, but anyway, when you press M4, it offers to install Armory and in parallel calls Helper

**Reply by seerge** [reply]
You need to run bat file as admin. And in general, I would recommend to read FAQ at https://github.com/seerge/g-helper it has all you main questions covered

**Reply by AreyUA** [reply]
Thanks a lot I'm reading it now

**Comment by AreyUA** [comment]
![image](https://github.com/seerge/g-helper/assets/126382491/0cc4df99-2850-40c0-8693-f220f5b5b4dd)
why  such a low fan speed?

**Reply by seerge** [reply]
@AreyUA fan speed is absolutely fine, it's just max RPM axis got misscalibrated (cause by some reason bios has reported that max RPM is 30 000 instead of usual 6000 or so) 

Open config.json at ``%appdata%\ghelper`` and remove line with ``fan_rpm`` . And then restart g-helper :)

**Comment by AreyUA** [comment]
I'm sorry, I can't find this file, not through the search, not manually

**Reply by seerge** [reply]
Just open ``%appdata%\ghelper`` :) (open explorer and copy that into address bar)

**Comment by AreyUA** [comment]
thanks deleted. now I'm looking at the scores

**Comment by AreyUA** [comment]
I just can’t understand why my processor is practically not loaded in the game, the video card too. The FPS is very small, other players have many times weaker hardware and there is a large FPS, for several months now I have been trying to find the reasons and I can’t ((((

**Reply by seerge** [reply]
@AreyUA I don't think it's related to the app at all :) Check your game / windows settings, if they actually use GPU, etc.
